pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -45,11 +50,6 @@ DC/OS 1.10.0 includes many new capabilities for Operators and expands the collec
   
 Please try out the new features and updated data services. Provide any feedback through our support channel: <a href="https://support.mesosphere.com/">support.mesosphere.com</a>.
 
-### Contents
-- [New Features and Capabilities](#new-features)
-- [Breaking Changes](#breaking-changes)
-- [Known Issues and Limitations](#known-issues)
-
 # <a name="new-features"></a>New Features and Capabilities
```

Comment:
Headings are formatted strangely. I think this should be `## New Features and Capabilities`

Reply:
Done.